src/test/java/Pages/loginPage.java

```diff
@@ -0,0 +1,22 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class loginPage extends basePage {
+    By emailFieldInput = By.cssSelector("[type = 'email']");
+    By passwordFieldInput = By.cssSelector("[type = 'password']");
+     By submitButton = By.cssSelector("button[type = 'submit']");
```

Comment:
If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```